R/descriptives.R

```diff
@@ -110,14 +111,20 @@ coef_var.default <- function(x, verbose = TRUE, ...) {
 #' @export
 coef_var.numeric <- function(x, mu = NULL, sigma = NULL,
                              method = c("standard", "unbiased", "median_mad", "qcd"),
-                             trim = 0, na.rm = FALSE, n = NULL, ...) {
+                             trim = 0, remove_na = FALSE, n = NULL, na.rm = FALSE, ...) {
+  # TODO: remove deprecated argument later
+  if (!missing(na.rm)) {
+    # TODO: add deprecation warning in a later update
+    remove_na <- na.rm
+  }
```

Comment:
why not add the deprecation message now? that way we can use `remove_na` in other easystats packages as soon as we see the message

Reply:
bump

Reply:
I prefer a "smoother" transition to deprecation, but I don't mind adding the warning now.

Reply:
We need a way to tell the users to use `remove_na` instead of `na.rm` so I guess the sooner the better